### DISCUSSION TITLE
Prevent Carousel from snapping to top of page

### DISCUSSION THREAD

**Discussion by munroe7** [saadeghi/daisyui#2977] [Q&A]
Pretty self explanitory, when you try and change slides in the carousel, it snaps the slide to the top of the window. Is there a way to prevent this? 

I really love this viewer but this is a huge deal breaker for me so I have to explore new options until I can find a resolution. e.preventDefault is not stopping this.

**Comment by saadeghi** [comment]
It should be possible using JS.

Search for `prevent anchor scroll` and there are many ways.

If you want more functionality and customizability for a carousel, I would suggest using a JS plugin for the carousel.  
It's completely fine to use daisyUI alongside any JS plugin because daisyUI carousel is basically just a scrollable area with scroll snap. JS carousel plugins on the other hand can control every event, positioning, transition effects etc.


Let me know if you have any questions.
